flybywiresim/aircraft
# fix(fms): fix T/D REACHED not showing in selected speed

Fixes #8261
Fixes #9338

## Summary of Changes

Fixes a logic error where T/D REACHED would only show in managed speed. This was likely a leftover from the old DECELERATE logic which only triggers in managed speed.

Also removes a managed speed check for the pause-at-tod function.

## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

- Fly to cruise
- Select a speed
- Check that T/D REACHED still shows up on the PFD
- Start another flight, fly to cruise, turn on Pause at T/D,
- Make sure the simulator pauses at the T/D.


## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo**, **flybywire-aircraft-a380-842 (4K)** or **flybywire-aircraft-a380-842 (8K)** download link at the bottom of the page
